openassessment/data.py

```diff
@@ -428,6 +428,50 @@ def _map_sudents_and_scorers_ids_to_usernames(cls, all_submission_information):
 
         return cls._map_anonymized_ids_to_usernames(student_ids + list(scorer_ids))
 
+    @classmethod
+    def _map_block_usage_keys_to_display_names(cls, course_id):
+        """
+        Fetches all course blocks and filters out those of them, whoose
+        category not equal to ``openassessment``. Then builds
+        mapping between block usage key string and block display name.
+
+        Args:
+            course_id (string or CourseLocator instance) - id of course
+            resourse
+        Returns:
+            dictionary, that contains mapping between block usage
+            keys (locations) and block display names.
+        """
+        # pylint: disable=import-error
+
+        from lms.djangoapps.course_blocks.api import get_course_blocks
+        from openedx.core.djangoapps.content.block_structure.transformers import BlockStructureTransformers
+
+        from xmodule.modulestore.django import modulestore
+
+        store = modulestore()
+        course_usage_key = store.make_course_usage_key(course_id)
+
+        blocks = get_course_blocks(None, course_usage_key, BlockStructureTransformers())
```

Comment:
@0x29a Can you add a comment here stating that we're passing `BlockStructureTransformers` without any transformers to avoid the requirement of passing a `user` to `get_course_blocks`.